### DISCUSSION TITLE
doris1.1.3exec_mem_limit  没有限制住单条sql 消耗内存

### DISCUSSION THREAD

**Discussion by heeegong** [apache/doris#14129] [A - General / Q&A]
exec_mem_limit=4294967296，但是实际大sql 还是占用了大量内存，导致节点挂掉
![image](https://user-images.githubusercontent.com/112262899/200819972-e1614004-a872-4347-b477-ebb6f71a7ed6.png)

**Comment by zy-kkk** [comment]
请执行一下dmesg -T，截个图看一下，
然后请问一下机器的内存是多少，机器是否只有doris_be在运行，还有be.conf里是否设置过mem_limit这项配置

**Reply by heeegong** [reply]
3台be，机器内存32G，有一台混部了observer，be.conf 没有mem_limit 参数，默认mem_limit=80%。操作select * from tbl order by time desc ,表300G数据，执行sql后三台be内存从个位数直接打满，挂掉。
![image](https://user-images.githubusercontent.com/112262899/200983052-b75de0f3-768e-46ea-9381-058c8a985655.png)
![image](https://user-images.githubusercontent.com/112262899/200984288-2f746c07-ad7f-4c9c-8ebb-c17f6df78a79.png)
